data/json/items/tools.json

```diff
@@ -5610,7 +5610,31 @@
     "category" : "weapons",
     "use_action": { "type": "knife" }
   },
-
+  {
+    "id": "glaive",
```

Comment:
Yes, remove it from both version, and also you can put a json comment (aka "\\" entry) with explanantion that lack of ```"use_action": knife ``` is intentional.